test/serializers.test.ts

```diff
@@ -66,7 +69,8 @@ describe('serializers', () => {
     })
 
     it('returns nothing when passed nothing', () => {
-      expect(serializers.res()).toBe(undefined)
+      const res = undefined
+      expect(serializers.res(res)).toBe(undefined)
```

Comment:
Previously, this tested that `serializers.res()` returned undefined. However, TypeScript complained that `serializers.res()` required an argument. By passing in undefined, this simulates the previous test case while satisfying the type-checking of TypeScript.

Reply:
I ran into this same issue [here](https://github.com/probot/probot/pull/650/files#diff-ec3e9580ab249d1f76a175cc2ddbb3fc) and came up with the same solution. Typescript doesn't like optional arguments. 😕 

Nit: could we simplify to just `expect(serializers.res(undefined)).toBe(undefined)`

Reply:
I will push up your suggested change. 👍